MotionPlanningEnv/cilinder.obj

```diff
@@ -0,0 +1,268 @@
+# Blender v3.2.2 OBJ File: 'untitled.blend'
+# www.blender.org
+mtllib cilinder.mtl
```

Comment:
Could you change this to be cylinder instead of the dutch spelling?

Reply:
The name is updated to cylinder